psi4/driver/driver_nbody.py

```diff
@@ -25,830 +25,1291 @@
 #
 # @END LICENSE
 #
-
+"""Plan, run, and assemble QC tasks to obtain many-body expansion and basis-set superposition error treatments.
+
+=============
+ManyBody Flow
+=============
+Bullet points are major actions
+Lines of dashes denote function calls
+e/d/dd=dg/g/h := energy, dipole, dipole derivative = dipole gradient, gradient, Hessian
+`mc_(frag, bas)` := a modelchem index, mc; indices of real fragments, frag; set(bas - frag) are indices of ghost fragments. see "intermediates_energy" in big table below for example.
+
```

Comment:
The concept of multichem vs. n-body levels should be explained here. Include an example of how they differ.